pkg/hive/manager.go

```diff
@@ -107,11 +106,11 @@ func NewFromConfig(log *logrus.Entry, _env env.Core, restConfig *rest.Config) (C
 	}, nil
 }
 
-func (hr *clusterManager) CreateNamespace(ctx context.Context) (*corev1.Namespace, error) {
+func (hr *clusterManager) CreateNamespace(ctx context.Context, docID string) (*corev1.Namespace, error) {
 	var namespaceName string
 	var namespace *corev1.Namespace
 	err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-		namespaceName = "aro-" + uuid.DefaultGenerator.Generate()
+		namespaceName = docID
```

Comment:
does that mean we will not using `aro` prefix as the log searching rule for hive namespaces?

Reply:
I was 50-50 between using just uuid or aro-uuid and reached out to Amber for inputs, I will make the change to add prefix aro-